lib/pymedphys/_streamlit/apps/wlutz/_filtering.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2020 Cancer Care Associates
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import datetime
+
+from pymedphys._imports import numpy as np
+from pymedphys._imports import streamlit as st
+
+
+def filtering_image_sets(to_be_filtered):
```

Comment:
filter_image_sets might be a better name.
I'm sensing a pattern of using a name that implies an ongoing activity rather than an action.
Is there a purpose for that, or is this copying some approach from "found code"?

Reply:
> I'm sensing a pattern of using a name that implies an ongoing activity rather than an action.

Yup, good call. It was subconscious and I suspect it may be because as I was writing it, it was "ongoing" per-se, as the user (me) was able to update the parameters within the streamlit app and the filtering was "ongoing".

But nevertheless the action of this function when called is not ongoing.